### PR TITLE
Enhancement: improve logging, move it from log.warnings to prints with ability to suppress

### PR DESCRIPTION
**Pull request by kbolashev**

### CODE REVIEW

**Comment by kbolashev** [comment]
[PyCaret client modifications](https://trello.com/c/CEcHxdy7/1552-pycaret-client-modifications)

File: dagshub/common/helpers.py [hunk 1]
```diff
@@ -50,85 +48,11 @@ def get_project_root(root):
...
-            with open(root / '.dvc' / '.gitignore', 'w') as config_gitignore:
-                config_gitignore.write(config.CONFIG_GITIGNORE)
-
-    print('Repository initialized!')
+def log_message(msg, logger=None):
+    """
+    Logs message to the info of the logger + prints, unless the printing was suppressed
+    """
+    if not config.quiet:
+        print(msg)
```
Comment (idonov8) [inline dagshub/common/helpers.py:56]: Why not `log.warning()`?
I remember @guysmoilov saying a python library shouldn't use `print()`

**Review by idonov8 — CHANGES_REQUESTED** [review]
looks **_Cool_**

File: dagshub/common/helpers.py (hunk 1 above)
Comment (kbolashev) [reply dagshub/common/helpers.py:56]: PyCaret are against spamming warning logs with informational messages, those should be kept for warnings.
This right now is basically a compromise where:
- We have the info logs that are actually info(rmational)
- They are also printed out by default
- If you pass DAGSHUB_QUIET you're not getting the prints, but still getting the info logs

Which is tbh the compromise that satisfies most of the requirements.
Guy's complaint about prints was inability to turn them off

Comment (kbolashev) [reply dagshub/common/helpers.py:56]: @idonov8

**Review by idonov8 — APPROVED** [review]
**_COOL 🐶_**

**Pull request merged**